virttest/utils_v2v.py

```diff
@@ -577,8 +577,11 @@ def get_vm_pci_list(self):
         """
         Get vm pci list.
         """
-        cmd = "lspci"
-        return self.run_cmd(cmd)[1]
+        cmd_list = ['lspci', 'lshw']
+        for cmd in cmd_list:
+            status, output = self.run_cmd(cmd)
```

Comment:
Do you need handle possible exception thrown by executed by 'lspci' ? If exception throw, lshw will never be executed

Reply:
I think it's not neccessary, lspci never throw out an exception. once it do happens, lshw will fail too because of same exception.